.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - cmd/protoc-gen-go-grpc/v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  create_release:
+    name: Create cmd/protoc-gen-go-grpc Release
+    runs-on: ubuntu-latest
+    outputs: 
+      upload_url: ${{ steps.create_release.outputs.upload_url }} 
+    
+    steps:
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          draft: true
+          prerelease: false
+
+  build_release:
+    name: Build Release assets
+    runs-on: ubuntu-latest
+    needs: create_release
+    strategy:
+      matrix:
+        goos: [linux, darwin, windows]
+        goarch: [386, amd64]
+        exclude:
+          - goos: darwin
+            goarch: 386
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
```

Comment:
Is there a way to specify "use the latest stable Go release version" instead of picking a version?

Reply:
Yes, without specifying the version, but I'd recommend against it. Using the latest version always carries a risk of a failing release. Updating the version twice a year shouldn't be too much effort, but I can remove it if you prefer using the latest version.

Reply:
I think I like this way (no version specified) better.  I'm not too worried about a failure here because of the Go version, and if that happens, which should be rare, we can do it manually.